src/liballoc/heap.rs

```diff
@@ -27,36 +27,46 @@ pub mod __core {
 
 extern "Rust" {
     #[allocator]
+    #[allocator_nounwind]
```

Comment:
Sure yeah, I can change the name (currently this is just allowed under "custom_attributes")